develop/dev-guide-bookshop-schema-design.md

```diff
@@ -135,78 +163,81 @@ WHERE table_schema LIKE 'bookshop';
+This table stores records of user ratings on books.
 
-| Field name | Type     | Explain                                                    |
+| Field name | Type     | Description                                                    |
 |------------|----------|------------------------------------------------------------|
-| book_id    | bigint   | Unique ID of the book (linked to [books](#books-table))    |
+| book_id    | bigint   | Unique ID of a book (linked to [books](#books-table))    |
 | user_id    | bigint   | User's unique identifier (linked to [users](#users-table)) |
 | score      | tinyint  | User rating (1-5)                                          |
 | rated_at   | datetime | Rating time                                                |
 
 ### `book_authors` table
 
-An author may write multiple books, and a book may require multiple authors to write at the same time. This table is used to store the correspondence between books and authors.
+An author may write multiple books, and a book may involve more than one author. This table stores the correspondence between books and authors.
 
-| Field name | Type       | Explain                                                      |
+| Field name | Type       | Description                                                      |
 |------------|------------|--------------------------------------------------------------|
-| book_id    | bigint(20) | Unique ID of the book (linked to [books](#books-table))      |
-| author_id  | bigint(20) | Unique ID of the author（Link to [authors](#authors-table)） |
+| book_id    | bigint(20) | Unique ID of a book (linked to [books](#books-table))      |
+| author_id  | bigint(20) | Unique ID of an author（Link to [authors](#authors-table)） |
 
 ### `orders` table
 
-This table is used to store order information for users to purchase books.
+This table stores user purchase information.
 
-| Field name | Type       | Explain                                                        |
+| Field name | Type       | Description                                                        |
 |------------|------------|----------------------------------------------------------------|
-| id         | bigint(20) | The unique ID of the order                                     |
-| book_id    | bigint(20) | Unique ID of the book (linked to [books](#books-table))        |
+| id         | bigint(20) | Unique ID of an order                                     |
+| book_id    | bigint(20) | Unique ID of a book (linked to [books](#books-table))        |
 | user_id    | bigint(20) | User unique identifier (associated with [users](#users-table)) |
 | quantity   | tinyint(4) | Purchase quantity                                              |
 | ordered_at | datetime   | Purchase time                                                  |
 
-## Database initialization script `dbinit.sql` 
+## Database initialization script `dbinit.sql`
```

Comment:
The new section description needs to be translated:

https://github.com/pingcap/docs-cn/blob/e02b2eab472ac67ab2a9d470dcf2e00447204cc8/develop/dev-guide-bookshop-schema-design.md#%E6%95%B0%E6%8D%AE%E5%BA%93%E5%88%9D%E5%A7%8B%E5%8C%96-dbinitsql-%E8%84%9A%E6%9C%AC